test/test_console_tool.py

```diff
@@ -580,6 +581,41 @@ def test_bad_terminal(self):
         console_tool = ConsoleTool(self.b2_api, stdout, stderr)
         console_tool.run_command(['b2', 'authorize_account', 'my-account', 'good-app-key'])
 
+    def test_clock_skew_hook_bad_format(self):
+        response = mock.MagicMock()
+        response.headers = {'Date': 'bad format'}
+        with self.assertRaises(BadDateFormat):
+            ClockSkewHook().post_request('POST', 'http://foo.com', {}, response)
```

Comment:
Use `example.com` or `example.org`, which are designated for this. `foo.com` is an actual domain that someone has, not sure if we want to associate with them.